dts/bindings/mipi-dbi/mipi-dbi-device.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+#
+# Common fields for MIPI-DBI devices
+
+include: [base.yaml, power.yaml]
+
+on-bus: mipi-dbi
+
+properties:
+  mipi-max-frequency:
+    type: int
+    description: Maximum clock frequency of device's MIPI interface in Hz
+  duplex:
```

Comment:
We might consider duplicating other SPI related properties such as `mipi-cpol`, `mipi-cpha`, `mipi-hold-cs`. At least, our LCD controller supports controlling all these three properties. If not, let me know, so I add them as vendor-specific properties.

Reply:
Ours supports CHA and CPOL in SPI mode as well. I'll add these properties, but I am going to create a `mipi-dbi-spi-device` binding for this, since these properties would not apply to other interfaces